GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -122,7 +122,14 @@ private FormBrowse()
             InitializeComplete();
         }
 
-        public FormBrowse([NotNull] GitUICommands commands, string filter, ObjectId selectCommit = null)
+        /// <summary>
+        /// Open Browse - main GUI including dashboard
+        /// </summary>
+        /// <param name="commands">commands in the current form</param>
+        /// <param name="filter">filter to apply to browse</param>
+        /// <param name="selectedId">Currently selected commit id</param>
+        /// <param name="firstId">First commit id in a diff</param>
```

Comment:
Should we call it `diffId` instead?

Reply:
(selected|first)(Id|Commit) is the common name, except in fileStatusList where First/Second is used (as Selected is not always Second). (Id for ObjectId and Commit for GitRevision).

"selected" is the primary, last selected commit which normally is the second in a diff (to or B)
"first" is the first in a diff, there can be several first before the selected though.